ci/configure/openshift-auth.sh

```diff
@@ -40,3 +40,5 @@ BRIDGE_K8S_MODE_OFF_CLUSTER_THANOS=$(oc -n openshift-config-managed get configma
 BRIDGE_K8S_MODE_OFF_CLUSTER_ALERTMANAGER=$(oc -n openshift-config-managed get configmap monitoring-shared-config -o jsonpath='{.data.alertmanagerPublicURL}')
 
 BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=${BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT:=$(oc whoami --show-server)}
+
+set +x
```

Comment:
remember to remove this when ready to merge

Reply:
I find the shell command echoing very distracting in general, and the `source ...` bit in the `start-console.sh` script has the side effect of carrying the `set -x` forward.  It needs to be turned off when control goes back to the `start-console.sh` script.

Reply:
then maybe we don't need the `-x` at the top ?

in any case, seeting something on the buttom of a file does not look good

Reply:
I refactored the file to be a set of functions instead.  Keeps things a bit tidier.